Aptoide/aptoide-client-v8
# [ASV-1270] Promotions Downgrade

**What does this PR do?**

   This PR handles the downgrade case for the install button on the promotions main view.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] PromotionsAdapter.java

**How should this be manually tested?**

Create an app with the same package name of any of the other apps in the list, but with a higher vercode. Install it on your device. Go to the promotions view, press install for that app and it should perform the downgrade (uninstall current version and install the new one).

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1270](https://aptoide.atlassian.net/browse/ASV-1270)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
